web/pkg/flagset/flagset.go

```diff
@@ -170,7 +170,7 @@ func ServerWithConfig(cfg *config.Config) []cli.Flag {
 		},
 		&cli.StringFlag{
 			Name:        "web-config-theme",
-			Value:       flags.OverrideDefaultString(cfg.Web.Config.Theme, "owncloud"),
+			Value:       flags.OverrideDefaultString(cfg.Web.Config.Theme, "https://localhost:9200/themes/owncloud/theme.json"),
```

Comment:
we at least need to use OCIS_URL in that case as a fallback URL. Are relative paths also supported?

Reply:
which actually gets quite ugly because OCIS_URL ist just expected to have the protocol+host part (eg. https://localhost:9200)

Reply:
What's your suggestion? The default is "check in the config file for web" which goes looking at whatever path is provided and if that one fails it loads the default theme we ship inside `web`...

Reply:
Just so I'm on the same page, the semantics of `$WEB_UI_CONFIG_THEME`, as I gather from the above discussion, is an endpoint where web downloads the theme? If that is the case we have to rename it to `$WEB_UI_CONFIG_THEME_URL`, since web expects a URL (in the strict definition of URL, which means no relative paths, since it is not a resource...)

Or else it looks to me that we have 2 config values to build the path that @wkloucek described above:

`"https://localhost:9200/themes/owncloud/theme.json"` is the combination of:

`{$WEB_UI_CONFIG_SERVER || $OCIS_URL}/themes/{$WEB_UI_CONFIG_THEME}/theme.json`

And both of them have the default set so the end result is a URL that @wkloucek described above. But these changes probably go beyond the scope of this ticket since it is docs-only 😸